helpers/deploymentConfig.ts

```diff
@@ -3066,7 +3066,7 @@ export const globalConfig: NetworkConfig = {
             supplyCap: convertToUnit(20_000, 18),
             borrowCap: convertToUnit(2_000, 18),
             reduceReservesBlockDelta: DEFAULT_REDUCE_RESERVES_BLOCK_DELTA,
-            vTokenReceiver: preconfiguredAddresses.sepolia.VTreasury,
+            vTokenReceiver: `0xF6C612c745Ba4546075DB62902c1Eb3255CdAe28`,
```

Comment:
`0xF6C612c745Ba4546075DB62902c1Eb3255CdAe28` is the receiver of the minted vTokens **only** for the weETH market